### PR TITLE
Prepare new release: ``v0.13.0``

### PR DESCRIPTION
**Pull request by behackl**
Preparations for `v0.13.0`.

### CODE REVIEW

File: docs/source/changelog/0.13.0-changelog.rst [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+release. People with a '+' by their names authored a patch for the first
+time.
+
+* Benjamin Hackl
+* Christopher Besch
+* Darylgolden
+* Filip +
+* Laith Bahodi
+* Lucas Ricci +
+* ManimBot +
```
Comment (kolibril13) [inline docs/source/changelog/0.13.0-changelog.rst:20]: do we want to include the manimbot?

**Comment by kolibril13** [comment]
https://manimce--2353.org.readthedocs.build/en/2353/changelog/0.13.0-changelog.html

File: docs/source/changelog/0.13.0-changelog.rst (hunk 1 above)
Comment (naveen521kk) [reply docs/source/changelog/0.13.0-changelog.rst:20]: I guess no, we should ignore like the pre-commit bot

Comment (behackl) [reply docs/source/changelog/0.13.0-changelog.rst]: I've adapted the changelog script.

**Pull request merged**